### ISSUE TITLE
Current Release Change outdated

### ISSUE THREAD

**Issue by cr1ogen** [qtile/qtile#3140]
Change the current release from qtile.org page to 19.0

**Comment by elParaguayo** [comment]
Website looks like it's a bit broken. That should update automatically, I think.

**Comment by m-col** [comment]
Though interestingly the release pages were built: http://docs.qtile.org/en/v0.19.0/

**Comment by m-col** [comment]
We should also set the default build to 'stable' (assuming that is the latest release) rather than 'latest'. Having the git version of the docs online confuses a lot of people running from a release.

**Comment by m-col** [comment]
@elParaguayo has changed the RTD settings to set `stable` as the default doc version. What's the difference between `stable` and `latest`? I guess `latest` will correspond to the latest commit, git version?

**Comment by elParaguayo** [comment]
Correct. Latest is current git commit.  Stable should be last tagged release.

**Comment by m-col** [comment]
Great, I'll close this then. Thanks for fixing!

**Comment by elParaguayo** [comment]
Ah. No. We're talking about qtile.org and not the docs.

**Comment by m-col** [comment]
Ah, interesting - what's up with that?

**Comment by elParaguayo** [comment]
The website gets it from here:
https://github.com/qtile/qtile.org/blob/5b5f5327e5a12218c58162b22cadc31f1b78e2e5/qtile/settings/common.py#L139-L145

Don't know why that's stopped working.

**Comment by m-col** [comment]
Do we know when it last was built?

**Comment by elParaguayo** [comment]
Last commit was before that release. I don't know if there's some hook to trigger a rebuild but I'm guessing that's the way. I don't have the login details... yet.
